repo/fsrepo_test.go

```diff
@@ -80,8 +80,9 @@ func TestFSRepoInit(t *testing.T) {
 
 	dir, err := ioutil.TempDir("", "")
 	assert.NoError(t, err)
-
-	defer os.RemoveAll(dir)
+	defer func() {
+		require.NoError(t, os.RemoveAll(dir))
+	}()
```

Comment:
It would be worth factoring this out to a function, but you don't have to do that here.

Reply:
Along the same lines I have a slight preference for putting this all on one lines but that's purely a style opinion.

Reply:
Ya, I think we can come back and make a good-first-issue to do this work.